Sources/Logging/LogHandler.swift

```diff
@@ -169,10 +169,10 @@ public protocol LogHandler: _SwiftLogSendableLogHandler {
     var logLevel: Logger.Level { get set }
 }
 
-extension LogHandler {
+public extension LogHandler {
```

Comment:
This is the inversion of our preferred style: we prefer to keep the extension undecorated and put visibility on the members. There's a lot of this change in the patch, can it be undone?